cmd/fossa/display/template.go

```diff
@@ -25,3 +37,20 @@ func Template(tmpl *template.Template, data interface{}) (string, error) {
 
 	return buf.String(), nil
 }
+
+// TemplateFormatTabs renders a template with the desired tab format.
+func TemplateFormatTabs(tmpl string, data interface{}, minWidth, tabWidth, padding int) (string, error) {
+	testTemplate, err := template.New("base").Parse(tmpl)
+	if err != nil {
+		return "", errors.Wrap(err, "Could not parse template data")
+	}
+
+	var buf bytes.Buffer
```

Comment:
please use [strings.Builder](https://godoc.org/strings#Builder) is output is only a string

Reply:
I know you didn't touch `func Template` immediately above, but it too should use `strings.Builder`